### DISCUSSION TITLE
AGG.

### DISCUSSION THREAD

**Discussion by gianlucaaiello56** [iisgiua/giuaschool#599] [Richieste di aiuto]
Salve, io ho eseguito l'aggiornamento per errore ed ora la schermata docenti ed alunni non funziona più. Si potrà risolvere con la versione 1.5.6. lunedì? @trinko

**Comment by Lucaccc** [comment]
> Salve, io ho eseguito l'aggiornamento per errore ed ora la schermata docenti ed alunni non funziona più. Si potrà risolvere con la versione 1.5.6. lunedì? @trinko

E' sufficiente che ripristini il backup del sito e del database a prima che installassi l'aggiornamento (sempre che tu l'abbia fatto), così tornerà come prima. Qualora non lo avessi fatto, il tuo hosting dovrebbe farne qualcuno in automatico, ma potrebbe accadere che sia di diverso tempo fa.

**Reply by gianlucaaiello56** [reply]
purtroppo anche facendo così non va, magari domani con la 1.5.6. si risolve il problema aggiornando ?
